proto/pulumi/language.proto

```diff
@@ -109,4 +112,14 @@ message InstallDependenciesRequest {
 message InstallDependenciesResponse {
     bytes stdout = 1; // a line of stdout text.
     bytes stderr = 2; // a line of stderr text.
+}
+
+message GenerateProjectRequest {
```

Comment:
This is a good place for doc comments on what it's supposed to do.

Reply:
Oh yeh fully plan on it, I should probably mark this PR as draft for now. Very happy to get comments on the direction, but it is still very WIP